README.md

```diff
@@ -4,6 +4,16 @@
 ```
 yarn
 ```
+## Environment
+Windows
+```
+set APP_ENV = testing
```

Comment:
The `APP_ENV` variable will select a particular `.env` file to use. The best default value is `APP_ENV=testnet`.

Reply:
Oh really sorry! missed that

Fixed this in a recent commit